src/Entity/IdeasWorkshop/Idea.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+namespace AppBundle\Entity\IdeasWorkshop;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\Committee;
+use AppBundle\Entity\EntityIdentityTrait;
+use AppBundle\Entity\EntityNameSlugTrait;
+use AppBundle\Entity\EntityTimestampableTrait;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity
+ *
+ * @ORM\Table(
+ *     name="ideas_workshop_idea",
+ *     uniqueConstraints={
+ *         @ORM\UniqueConstraint(name="idea_name_unique", columns="name"),
```

Comment:
same question for name unique constraint, I think it will be enough on slug column